test/smoke/package/src/helpers/testApplicationSetupManager.ts

```diff
@@ -0,0 +1,498 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as path from "path";
+import * as fs from "fs";
+import * as semver from "semver";
+import * as rimraf from "rimraf";
+import * as utilities from "./utilities";
+import { SmokeTestsConstants } from "./smokeTestsConstants";
+import { SmokeTestLogger } from "./smokeTestLogger";
+import { vscodeManager } from "../main";
+
+export class TestApplicationSetupManager {
+
+    private cacheDirectory: string;
+    private testAppsDirectory: string;
+
+    private rnWorkspaceDirectory: string;
```

Comment:
My opinion that it is better to create interfaces for this, so these strings will be gathered in one object.